charts/opentelemetry-collector/templates/ingress.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
   {{- else }}
   name: {{ include "opentelemetry-collector.fullname" $ }}
   {{- end }}
+  namespace: {{ template "opentelemetry-collector.namespace" . }}
```

Comment:
@krisztianfekete I believe this is causing your issue.  The ingress template includes a `range` at the top which changes the scope that everything inside works on.  `.` in this instance is not referencing the entire configuration object but a specific subset that does not include `namespaceOverride`.  You'll need to do some manipulation or use `$` (I think, you'll need to look it up) to reference the outer scope.  Also change the other files you change and make sure `.` is the appropriate scope.

Reply:
Excellent, that was it! Thanks @TylerHelmuth. I rebased it too, so now it should be ready to merge!